app/src/utils/ListPlaceholder.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Dimensions, View } from 'react-native';
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder';
+
+interface ListPlaceholderProps {
+  placeholderCount: number;
+}
+
+export const ListPlaceholder: React.FC<ListPlaceholderProps> = ({
+  placeholderCount,
+}) => {
+  const placeholderWidth = Dimensions.get('screen').width - 40;
+
+  return (
+    <SkeletonPlaceholder>
+      <View
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}>
+        {[...Array(placeholderCount)].map((index) => (

```

Comment:
Patrząc od strony performance to lepiej byłoby wygenerować tą listę z View raz i wpakować w useMemo